aioopenssl/__init__.py

```diff
@@ -632,6 +641,10 @@ def starttls(self, ssl_context=None,
         if post_handshake_callback is not None:
             self._tls_post_handshake_callback = post_handshake_callback
 
+        # Drain before initializing TLS
+        while self._buffer:
+            yield from asyncio.sleep(0)
```

Comment:
I haven’t looked at this into detail yet, so this may be stupid, but yield from self.drain() is no option?

Also, this makes concurrent writes undefined, which should at least be documented.

In addition, a very simple server-mode test (possibly without starttls) should be added. I’m trying to get some unit-test coverage into aioopenssl.